### PR TITLE
Add file to window title and improve viewport navigation.

### PR DESCRIPTION
**Pull request by Kiisu-Master**
- Add the currently open file path to the window title.
- Add ctrl + middle mouse zooming.
- Add comments to viewport input handling.
- Change viewport input handling to be else-if cases for event types.

### CODE REVIEW

File: src/ui_parts/viewport.gd [hunk 1]
```diff
@@ -55,22 +58,44 @@ func _unhandled_input(event: InputEvent) -> void:
...
+		else:
+			set_view(view.position - (wrap_mouse(event.relative)\
+					if GlobalSettings.wrap_mouse else event.relative) / Indications.zoom)
 	
-	if event is InputEventPanGesture:
+	elif event is InputEventPanGesture:
+		
+		# Zooming with ctrl + touch?
 		if event.ctrl_pressed:
 			zoom_menu.set_zoom(Indications.zoom * (1 + event.delta.y / 2))
```
Comment (Kiisu-Master) [inline src/ui_parts/viewport.gd:82]: Idk what this is

**Comment by MewPurPur** [comment]
Since the logic is under InputEventMouseMotion, can you do this with one of the properties for checking for relative movement instead of with a variable?

**Comment by Kiisu-Master** [comment]
The variable is storing the position it will zoom to.
I don't see a property in the event that contains the starting position, so i dont think i could remove it.
This is not a relative thing. Zooming amount is using the relative movement, also with warping, but there is no way to know the position to zoom to in a stateless way.

**Comment by MewPurPur** [comment]
Thanks!

**Pull request merged**